lib/matplotlib/transforms.py

```diff
@@ -2557,9 +2560,11 @@ def depth(self):
         return self._a.depth + self._b.depth
 
     def _iter_break_from_left_to_right(self):
-        for lh_compliment, rh_compliment in self._a._iter_break_from_left_to_right():
+        for lh_compliment, rh_compliment \
+                in self._a._iter_break_from_left_to_right():
             yield lh_compliment, rh_compliment + self._b
-        for lh_compliment, rh_compliment in self._b._iter_break_from_left_to_right():
+        for lh_compliment, rh_compliment \
+                in self._b._iter_break_from_left_to_right():
```

Comment:
I'm not a fan of any of the changes you are making here in transforms.  We have long had a style policy of avoiding trailing backslashes.  I don't see anything wrong with the original versions of these lines.  Is this file excluded from PEP8-checking because the lines are too long?  If so, shorten by changing 'lh_compliment' to 'left', etc.  Or just leave the poor file in peace.  (I suspect 'lh_compliment' should actually be spelled 'lh_complement' anyway; but 'left, right' is probably easier to read and no less clear.)

Reply:
Yes, the lines were too long (and while a strict 79 character limit may be overly strict, there is a real benefit in keeping lines short -- not everyone has a wide screen).
But I agree that renaming the variables is better, fixed.